### ISSUE TITLE
No section in debian control

### ISSUE THREAD

**Issue by 532910** [RustScan/RustScan#114]
`Section` must be filled to include in apt repository:

```
# reprepro includedeb sid rustscan_1.4.0_amd64.deb
No section given for 'rustscan', skipping.
```

https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-section
https://www.debian.org/doc/debian-policy/ch-archive.html#s-subsections

**Comment by brandonskerritt** [comment]
Hey! Thanks for pointing this out, I'll release the new `.deb`s later on today. They'll have this metadata:

```
[package.metadata.deb]
depends = "$auto, nmap"
section = "rust"
```

**Comment by brandonskerritt** [comment]
This is now fixed in 1.5.2, which is being uploaded right now. :)

**Comment by 532910** [comment]
Thank you, Autumn.
